pkg/catalog/testdata/broken-chart-ref-environment/environments/testing/releases/test-release.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: joy.nesto.ca/v1alpha1
+kind: Release
+metadata:
+  name: test-release
+  annotations:
+    nesto.ca/notifications-channel: notif-cicd-cross-system
+spec:
+  project: testing
+  version: 0.0.1
+  chart:
+    ref: missing-ref
+  values:
+    common:
+      labels:
+        nesto.ca/stream: platform
+    api:
+      enabled: true
+      container:
+        port: 3000
+        livenessProbe:
+          httpGet:
+            path: /napi/healthz
+            port: 3000
+      deployment:
+        replicas: 1
+      service:
+        targetPort: 3000
```

Comment:
Maybe strip away all text that possibly can be removed in all files to simplify and emphasize objective of test?